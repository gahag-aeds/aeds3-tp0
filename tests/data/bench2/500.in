500 250
63 76 35 21 95 8 18 88 27 60 61 38 73 78 72 17 99 36 58 62 0 82 24 2 79 43 12 55 57 83 20 73 59 7 94 6 15 12 95 94 25 8 85 50 38 57 67 37 93 77 51 46 11 75 48 90 70 12 45 28 47 17 1 59 76 47 65 91 60 12 38 37 20 23 87 10 80 7 99 26 36 2 72 48 78 72 38 0 36 36 80 84 5 33 95 82 81 60 25 93 73 63 30 45 86 17 8 19 76 7 45 13 10 69 61 40 41 51 40 77 87 73 13 45 6 8 27 87 21 52 80 46 68 62 43 54 32 51 25 8 59 70 73 21 39 34 61 32 38 53 10 77 26 75 22 85 36 1 72 57 6 5 55 74 67 98 80 99 50 6 60 61 76 33 82 68 20 95 0 58 0 62 35 79 90 10 64 26 63 88 35 69 93 90 95 61 88 76 12 38 82 72 51 10 6 85 30 78 32 83 36 33 45 23 64 35 85 28 61 49 16 96 18 62 86 14 75 27 42 87 17 76 60 69 38 18 6 69 96 39 52 84 24 49 59 88 37 45 68 98 94 36 47 64 98 85 30 73 64 24 61 82 0 73 3 39 43 9 60 91 0 64 75 24 13 34 64 50 79 84 1 25 21 0 42 19 37 72 93 2 49 6 84 49 79 39 88 74 48 0 65 49 64 40 25 78 26 90 80 58 74 81 83 47 33 25 67 71 50 12 73 99 18 9 0 49 48 41 23 48 41 88 49 6 80 75 36 6 17 68 64 43 50 0 91 35 77 58 6 79 22 31 78 40 92 31 89 92 72 12 41 13 52 90 71 32 17 59 90 34 28 7 30 30 7 21 65 84 31 72 64 53 55 94 45 48 77 34 40 49 98 33 15 50 76 38 34 93 98 76 80 78 83 62 8 42 83 25 79 66 49 95 71 5 41 16 5 19 2 97 68 0 83 35 2 59 74 88 4 24 16 36 2 52 98 62 94 33 39 25 51 89 20 22 46 62 90 51 33 92 0 1 44 83 89 98 42 63 38 99 39 55 35 93 7 86 55 53 19 94 31 71 35 51 45 81 65 36 32 98 80 33 52 77 68 41 27 11 56 66 62 95 21 49 88 80 35 95 85 55 
Sub 14 179
Max 9 174
Add 271 436
Max 84 249
Add 259 424
Min 184 349
Add 249 414
Max 38 203
Sub 166 331
Max 36 201
Add 239 404
Sum 39 204
Add 95 260
Sum 19 184
Sub 264 429
Max 47 212
Sub 52 217
Max 329 494
Sub 144 309
Max 14 179
Add 143 308
Max 256 421
Add 31 196
Max 198 363
Sub 272 437
Min 127 292
Sub 120 285
Sum 287 452
Sub 192 357
Max 300 465
Sub 328 493
Sum 205 370
Sub 308 473
Sum 264 429
Sub 34 199
Min 312 477
Sub 194 359
Min 29 194
Sub 96 261
Sum 105 270
Sub 64 229
Sum 334 499
Add 223 388
Max 238 403
Sub 98 263
Min 152 317
Sub 30 195
Max 149 314
Add 147 312
Sum 254 419
Sub 242 407
Sum 35 200
Add 175 340
Max 272 437
Add 233 398
Max 112 277
Sub 144 309
Sum 18 183
Add 65 230
Sum 1 166
Add 63 228
Sum 58 223
Add 285 450
Sum 117 282
Sub 68 233
Max 319 484
Sub 174 339
Sum 7 172
Add 13 178
Min 269 434
Add 111 276
Min 156 321
Sub 14 179
Max 334 499
Add 139 304
Sum 111 276
Add 231 396
Min 248 413
Add 5 170
Max 43 208
Add 167 332
Max 246 411
Sub 158 323
Min 8 173
Add 263 428
Min 56 221
Add 119 284
Sum 72 237
Add 153 318
Min 263 428
Sub 152 317
Sum 216 381
Add 67 232
Sum 103 268
Sub 78 243
Max 134 299
Add 167 332
Sum 251 416
Sub 140 305
Sum 179 344
Sub 186 351
Sum 330 495
Sub 80 245
Max 279 444
Add 75 240
Min 298 463
Sub 56 221
Max 306 471
Add 211 376
Sum 60 225
Sub 94 259
Min 44 209
Sub 306 471
Sum 251 416
Add 51 216
Max 234 399
Sub 306 471
Sum 249 414
Sub 52 217
Max 205 370
Sub 258 423
Max 203 368
Sub 166 331
Sum 324 489
Add 305 470
Min 243 408
Add 203 368
Max 137 302
Add 159 324
Sum 88 253
Add 61 226
Min 90 255
Add 83 248
Sum 220 385
Add 35 200
Min 158 323
Add 183 348
Max 170 335
Add 129 294
Min 60 225
Add 229 394
Max 222 387
Add 183 348
Sum 280 445
Add 219 384
Max 233 398
Sub 260 425
Sum 270 435
Add 227 392
Max 311 476
Sub 140 305
Sum 150 315
Add 179 344
Max 305 470
Add 219 384
Max 230 395
Add 213 378
Min 87 252
Sub 32 197
Sum 118 283
Sub 174 339
Sum 173 338
Add 287 452
Sum 3 168
Sub 58 223
Min 67 232
Sub 160 325
Sum 321 486
Sub 236 401
Max 34 199
Sub 46 211
Min 211 376
Add 1 166
Max 308 473
Sub 236 401
Min 299 464
Sub 286 451
Sum 161 326
Add 235 400
Max 178 343
Add 137 302
Min 40 205
Sub 74 239
Max 62 227
Add 11 176
Max 292 457
Add 291 456
Max 303 468
Add 125 290
Sum 68 233
Sub 166 331
Min 44 209
Sub 240 405
Min 199 364
Sub 46 211
Max 44 209
Add 11 176
Sum 285 450
Sub 30 195
Max 327 492
Sub 64 229
Min 155 320
Add 203 368
Min 145 310
Sub 208 373
Max 104 269
Add 109 274
Sum 159 324
Add 11 176
Min 9 174
Sub 2 167
Min 227 392
Add 49 214
Min 155 320
Sub 288 453
Max 139 304
Add 193 358
Min 244 409
Add 187 352
Min 317 482
Sub 58 223
Min 99 264
Sub 26 191
Min 43 208
Add 297 462
Max 152 317
Sub 86 251
Max 307 472
Sub 102 267
Sum 195 360
Add 299 464
Sum 166 331
Add 95 260
Max 247 412
Sub 310 475
Sum 302 467
Add 97 262
Sum 84 249
Sub 206 371
Max 187 352
Add 173 338
Max 288 453
Add 79 244
Sum 221 386
Sub 108 273
Min 33 198
Add 105 270
Max 300 465
Sub 22 187
Sum 37 202
