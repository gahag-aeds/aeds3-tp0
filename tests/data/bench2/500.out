98
98
1
98
98
8162
7882
98
101
98
101
101
-1
8469
100
8230
8029
-2
-3
7891
8405
98
-6
96
8197
7480
98
98
7204
7452
7564
8015
100
7479
-1
-5
99
8009
1
97
101
-2
-5
7861
0
8650
8277
100
8922
8560
8648
101
-2
101
7658
-6
8772
101
8776
100
99
8106
0
99
7861
-5
8679
-3
102
-2
103
8609
105
8680
105
9232
106
107
-2
8954
9447
7789
-5
8819
97
7
106
-2
8996
105
-6
99
105
106
7987
-7
2
96
8862
99
-10
-10
105
9129
-3
5
-8
106
4
-3
-9
-10
108
106
9429
9194
108
8889
8050
108
106
9624
-9
106
7580
